echopype/echodata/combine.py

```diff
@@ -24,6 +25,7 @@
 ED_GROUP = "echodata_group"
 ED_FILENAME = "echodata_filename"
 FILENAMES = "filenames"
+ECHODATA_FILENAME = "echodata_filename"
```

Comment:
Seems that you can just use `ED_FILENAME` for the new use of `ECHODATA_FILENAME`?

Reply:
Thanks for catching that 😅